org.eclipse.xpect/META-INF/MANIFEST.MF

```diff
@@ -25,7 +25,8 @@ Require-Bundle: org.apache.log4j;bundle-version="1.2.0";visibility:=reexport,
  org.eclipse.xtext.ecore;bundle-version="2.9.2";resolution:=optional,
  org.objectweb.asm;bundle-version="[3.0.0,8.0.0)";resolution:=optional,
  org.eclipse.equinox.common;bundle-version="3.0.0";resolution:=optional,
- org.eclipse.core.runtime;bundle-version="3.0.0";resolution:=optional
```

Comment:
i wonder if this should be added to the pom as dependency too for standalone calling from maven/gradle

Reply:
@dhuebner how do the maven artifacts you published deal with that issue? do they assume asm, equinox and others come from somewhere else/are added by the user?

Reply:
When using maven, isn't this handled by tycho?

Reply:
I just sag that the type id still eclipse bundle so the User has to take care about That itself (Same as for mwe)

Reply:
Sorry I didn't get that.

Reply:
the maven artifact type is eclipse-bundle => if you use pure gradle or maven then you have to care about transtive dependencies yourself as we do it for mwe

Reply:
in mwe we do this. but i saw its not done in xpect in maven: https://github.com/eclipse/mwe/blob/9adfa2372a1d0d562734fbc8d9679357730b029e/plugins/org.eclipse.emf.mwe2.launch/pom.xml#L18

Reply:
this is a general problem and not related to this specific pr. added a comment here: https://github.com/eclipse/Xpect/issues/267#issuecomment-504875500